shared/src/business/useCases/getPractitionerCasesInteractor.ts

```diff
@@ -30,12 +32,22 @@ export const getPractitionerCasesInteractor = async (
     .getPersistenceGateway()
     .getCasesByDocketNumbers({ applicationContext, docketNumbers });
 
-  cases.forEach(
-    aCase => (aCase.caseTitle = Case.getCaseTitle(aCase.caseCaption)),
-  );
+  const caseDetails: PractitionerCaseDetail[] = cases.map(c => {
+    c = formatCase(applicationContext, c, authorizedUser);
+    return {
+      caseTitle: c.caseTitle,
+      consolidatedIconTooltipText: c.consolidatedIconTooltipText,
+      docketNumber: c.docketNumber,
+      docketNumberWithSuffix: c.docketNumberWithSuffix,
+      inConsolidatedGroup: c.inConsolidatedGroup,
+      isLeadCase: c.isLeadCase,
+      isSealed: c.isSealed,
+      status: c.status,
+    };
+  });
```

Comment:
Before, we were returning a whole "case" object. (I put "case" in quotes because the available properties on a case differ depending on where it is at.) I have restricted this for two reasons:

1. This prevents a 502 error when there are a lot of cases. See [troubleshooting doc](https://github.com/ustaxcourt/ef-cms/blob/staging/docs/additional-resources/troubleshooting.md#502-errors).
2. This makes exactly what we have/need for a practitioner case explicit.

Reply:
An alternative approach would be to paginate here on the backend, but that would introduce further complexities, especially if we wanted to avoid re-sorting to get the right records on every page change.